widget/popup.go

```diff
@@ -241,5 +242,9 @@ func (r *modalPopUpRenderer) Refresh() {
 	if r.background.Size() != r.popUp.innerSize {
 		r.Layout(r.popUp.Size())
 	}
+	if !r.popUp.Canvas.Size().Subtract(r.popUp.BaseWidget.Size()).IsZero() {
+		r.popUp.BaseWidget.Resize(r.popUp.Canvas.Size())
+	}
+	r.popUp.Content.Refresh()
```

Comment:
Why is this refresh needed?
That invalidates all the caches for the whole dialog content which could be quite expensive.

Reply:
In my understanding, if something in a parent widget change, it should updates its children. This refresh guarantees the popup updates all its Content if theme changes.

Reply:
The theme application is done at the driver level and it forces all widgets to refresh.
A widget should only be refreshed if you know that its content must be re-drawn.

If the only change is the resize then it will cause a refresh on the elements that need to refresh on resize will do so themselves.

Reply:
I see, this was not visible when theme applied so was not updated.
OK